### ISSUE TITLE
Payara Server 4.1.1.161 log configuration issue

### ISSUE THREAD

**Issue by simonemazzoni** [payara/Payara#664]
I upgraded my AS from Payara Server 4.1.1.153 to Payara Server 4.1.1.161.
Everything seems to work fine except for a strange behavior regarding logging level.

In my project I use a library to manage url-rewriting called "Rewrite/Pretty Faces" (http://www.ocpsoft.org/rewrite/)

This library uses `org.apache.commons.logging.Log;` to log messages with various level of logging (INFO,DEBUG,ERROR mostly). In Payara Server 4.1.1.153 everything works fine and when I run my project in Production mode, log messages printed with DEBUG level are not printed to the console or server.log, whereas if I run the same project with the same configurations on Payara Server 4.1.1.161 all the DEBUG messages are printed to console and server log even if the project is run in Production mode.

I guess something changed on the logging default configuration with the new version, but I can't understand why. This is not exactly a bug since everything works correctly, but it is a bit annoying when I have to read the logs or the console to check for errors.

[https://github.com/ocpsoft/rewrite/blob/master/config-prettyfaces/src/main/java/com/ocpsoft/pretty/faces2/application/PrettyNavigationHandler.java#L60](url) here you can find the source code of the class that produces the log messages (at line 60).

The issue can be reproduced compiling and running this maven project on Payara Server 4.1.1.161 [https://github.com/ocpsoft/rewrite/tree/master/showcase/bookstore](url)

By checking the console log or server log you can see a lot of unnecessary row printed. If you execute the same project on Payara Server 4.1.1.153 this do not happen.

Thanks for the help.
Simone

**Comment by smillidge** [comment]
I think this is a duplicate of #652

**Comment by simonemazzoni** [comment]
I tried to set the pax-logger level to WARN as suggested in #652, using the JVM Option `-Dorg.ops4j.pax.logging.DefaultServiceLog.level=WARN`, but nothing has changed.

Am I missing something?

**Comment by simonemazzoni** [comment]
I noticed that on Payara Server 4.1.1.161 I have a strange error regarding the logging system.
Here the message found in console during the running process of my web app.

The relevant row is bolded.

```
Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishRuntime@65ae52e2 in service registry.
**Cannot find the resource bundle for the name com.sun.logging.enterprise.system.container.ejb for class org.glassfish.kernel.javaee.MEJBService using org.glassfish.main.core.javaee-kernel [134]**
#!## LogManagerService.postConstruct : rootFolder=/Applications/NetBeans/payara41_161/glassfish
#!## LogManagerService.postConstruct : templateDir=/Applications/NetBeans/payara41_161/glassfish/lib/templates
#!## LogManagerService.postConstruct : src=/Applications/NetBeans/payara41_161/glassfish/lib/templates/logging.properties
#!## LogManagerService.postConstruct : dest=/Applications/NetBeans/payara41_161/glassfish/domains/domain1/config/logging.properties
```

This error is not present if I user Payara Server 4.1.1.153.

**Comment by smillidge** [comment]
Please try the latest Jenkins Build as we believe we have squashed the logging problems

**Comment by simonemazzoni** [comment]
Jenkins Build fixed the issue.

When will be released the final build?

**Comment by smillidge** [comment]
Officially it will be end of May but I think we will do a patch build for this issue.

**Comment by simonemazzoni** [comment]
Ok thanks, I will wait for the patch.

**Comment by dorpsidioot** [comment]
That would be great, because this is stopping us from using the latest build in production.

**Comment by smillidge** [comment]
Resolved in 4.1.1.161.1
